src/solvers/sat/satcheck_ipasir.cpp

```diff
@@ -0,0 +1,200 @@
+/*******************************************************************\
+
+Module:
+
+Author: Norbert Manthey, nmanthey@amazon.com
+
+\*******************************************************************/
+
+#ifndef _MSC_VER
+#include <inttypes.h>
+#endif
+
+#include <cassert>
+#include <stack>
+
+#include <util/threeval.h>
+
+#include "satcheck_ipasir.h"
+
+#ifdef HAVE_IPASIR
+
+extern "C"
+{
+#include <ipasir.h>
+}
+
+/* solver interface:
```

Comment:
I'm not sure this should be documented here.
Maybe add a link to the full documentation if it exists.
Otherwise it can stay.